### PR TITLE
340 Disable 'show all' when there are no populations

### PR DESCRIPTION
**Pull request by Aiga115**
Issue #340 
Problem: Disable 'show all' when there are no populations
Solution: Added function which checks if population children have cells or not, in case if they don't function returns true and making switch disabled

image
image

### CODE REVIEW

File: applications/portal/frontend/src/components/sidebar/PopulationsAccordion.jsx [hunk 1]
```diff
@@ -128,6 +129,7 @@ const PopulationsAccordion = ({
                                 <Switch />
                             }
                             onChange={handleShowAllPopulations}
+                            disabled={areAllPopulationsWithChildrenCells(populations)}
```
Comment (afonsobspinto) [inline applications/portal/frontend/src/components/sidebar/PopulationsAccordion.jsx:132]: Why can't we just check if populations is empty?

File: applications/portal/frontend/src/components/sidebar/PopulationsAccordion.jsx [hunk 2]
```diff
@@ -94,6 +94,7 @@ const PopulationsAccordion = ({
         })
     }
 
+    console.log('populations: ', populations)
```
Comment (afonsobspinto) [inline applications/portal/frontend/src/components/sidebar/PopulationsAccordion.jsx:97]: Can be removed?

File: applications/portal/frontend/src/components/sidebar/PopulationsAccordion.jsx (hunk 1 above)
Comment (afonsobspinto) [reply applications/portal/frontend/src/components/sidebar/PopulationsAccordion.jsx:132]: image

**Review by afonsobspinto — CHANGES_REQUESTED** [review]

File: applications/portal/frontend/src/components/sidebar/PopulationsAccordion.jsx (hunk 1 above)
Comment (Aiga115) [reply applications/portal/frontend/src/components/sidebar/PopulationsAccordion.jsx:132]: @afonsobspinto what about if the case when all the children switches are disabled?

**Review by afonsobspinto — APPROVED** [review]

**Pull request merged**